tracing-attributes/src/attr.rs

```diff
@@ -177,9 +177,10 @@ impl Parse for EventArgs {
                     match ident.to_string().as_str() {
                         "Debug" => result.mode = FormatMode::Debug,
                         "Display" => result.mode = FormatMode::Display,
+                        "Raw" => result.mode = FormatMode::Raw,
```

Comment:
I don't love the name "Raw" for this mode --- it's not super obvious to the user what this actually means. Since the error is being recorded as a `dyn Error` trait object, and the other two format modes are named "Debug" and "Display", it might make more sense for this to be "Error" instead?

Reply:
ah, after taking a closer look, I see that this also controls how `ret` values, which are not errors, may be recorded. Maybe `Value` is a better name?

Reply:
I like `Value` here, but I also wonder whether we need to maybe subdivide this a bit more. For context, this PR is really introducing two things:

1. An ability to record the `err` and `ret` value attributes "in their raw value form" instead of relying on `std::fmt::*` for coercion, which resolves issues with how OpenTelemetry handles `Error` internally.
2. Adding event messages to represent the `err` and `ret` values so that they appear nicely in practical production UIs like Honeycomb/Jaeger etc.

The second of those points currently is always using `Display` semantics, however there are types where that is not implemented and `Debug` would be preferable. In those scenarios we may want to be able to specify `Value+Debug` and `Value+Display` to control these behaviours, or alternatively treat "log the attribute value in its raw form" as an option separate from the formatting of the messages.

I'll update this to use `Value` for now and we can discuss any other changes (either here or in a future PR depending on what you'd prefer).